internal/generator/vector/normalize.go

```diff
@@ -34,9 +34,11 @@ if !exists(.level) {
   }
 }
 `
-	RemoveSourceType = `del(.source_type)`
-	RemoveStream     = `del(.stream)`
-	RemovePodIPs     = `del(.kubernetes.pod_ips)`
+	RemoveSourceType   = `del(.source_type)`
+	RemoveStream       = `del(.stream)`
+	RemovePodIPs       = `del(.kubernetes.pod_ips)`
+	RemoveNodeLabels   = `del(.kubernetes.node_labels)`
```

Comment:
Why are we removing these maps?

Reply:
Because these were added since vector 0.21 and with them we get test failures like this:
```
[Functional][LogForwarding][Normalization] tests for message format
/home/blah/go/src/github.com/openshift/cluster-logging-operator/test/functional/normalization/message_format_test.go:16
  should parse application log format correctly if log message contains 'stdout','stderr' (Bug 1889595) [It]
  /home/blah/go/src/github.com/openshift/cluster-logging-operator/test/functional/normalization/message_format_test.go:87

  Expected no errors reading the logs
  Expected
      <*errors.errorString | 0xc000d832f0>: {
          s: "json: unknown field \"timestamp_end\"",
      }
  to be nil

  /home/blah/go/src/github.com/openshift/cluster-logging-operator/test/functional/normalization/message_format_test.go:105

```
due to strict log parsing. The types from https://github.com/openshift/cluster-logging-operator/blob/master/test/helpers/types/types.go define what fields we expect to find in the log events.